src/app/watchCourse/[watchCourseID]/page.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import WatchCourse from "../../../Components/WatchCourse/WatchCourse";
+
+const WatchCoursePage = ({ params }: { params: { watchCourseID: string } }) => {
+  return (
+    <>
+      <div>Watch Course Page {params.watchCourseID}</div>
```

Comment:
Watch Course için yeni bir route oluşturmamıza gerek yok. Bunun yerine aslında kullanıcı eğer kursu satın aldıysa singleCourse sayfasında yani kursun sayfasında kullanıcıya watchCourse componentlerini gösterebiliriz. Bunun için SingleCourse sayfasına frontend tarafı için bir değerde kursu satın alma durumu tutularak ona göre farklı componentleri gösterme mantığı entegre edebiliriz.